server/models/recipe.js

```diff
@@ -0,0 +1,51 @@
+import mongoose from 'mongoose'
+import slugs from 'slugs'
+
+const recipeSchema = new mongoose.Schema({
+
+  title: {
+    type: String,
+    required: true
+  },
+  slug: String,
+  image: {
+    type: String,
+    required: true
+  },
+  author: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  ingredients: {
+    type: [String],
+    required: false
+  },
+  steps: {
+    type: [String],
+    required: false
+  },
+  category: {
+    type: String,
+    required: true
+  },
```

Comment:
Category, as well as likes and author, should reference a seperate collection for where there information should be stored, since we will need to know a lot of information about them.